backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/numa/vm/NumaValidator.java

```diff
@@ -329,11 +329,6 @@ public ValidationResult checkVmNumaNodesIntegrity(final VM vm, final List<VmNuma
             return validationResult;
         }
 
-        // Check if the VM is pinned to at least one host
-        if (vm.getDedicatedVmForVdsList().isEmpty() && !VmCpuCountHelper.isResizeAndPinPolicy(vm)) {
-            return new ValidationResult(EngineMessage.ACTION_TYPE_FAILED_VM_NOT_PINNED_TO_AT_LEAST_ONE_HOST);
-        }
```

Comment:
why dropping it altogether instead of checking if there's NUMA pinning here?

Reply:
> it will fail on scheduler, it is possible also to validate here before.

you mean that we can skip failing here because it will fail later on when scheduling the vm? if we end up with a configuration that would never be valid, it's better to fail early